pkgs/tools/misc/oh-my-posh/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "oh-my-posh";
+  version = "8.1.0";
+
+  src = fetchFromGitHub {
+    owner = "JanDeDobbeleer";
+    repo = "oh-my-posh";
+    rev = "v${version}";
+    sha256 = "DH+Sh95nZVuNxldtBII+sFJzMGk82a5i3RgPSrw4Bhg=";
+  };
+  modRoot = "./src";
+  vendorSha256 = "iBVehyfJ2WX/nSBbXeOD0ozjLSVn5LKDNgbYVopIr4w=";
+
+  ldflags = [ "-a" "-s" "-w" "-X" "main.Version=${version}" "-extldflags" ''"-static"'' ];
+
+  tags = [ "netgo" "osusergo" "static_build" ];
+
+  postInstall = ''
+    mkdir -p $out
+    mkdir -p $out/share/oh-my-posh
+    cp -r ${src}/themes $out/share/oh-my-posh/
+  '';
+
+  meta = with lib; {
+    description = "A prompt theme engine for any shell";
+    homepage = "https://ohmyposh.dev/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ shrykewindgrace ];
+    # it should on darwin, too, but I do not have the expertise to make it work on nix
+    platforms = platforms.linux;
```

```suggestion
    platforms = platforms.darwin;
```
Maybe add a short comment about the error.

Also please squash all package commits into ``oh-my-posh: init at 8.1.0``.

---

Yeah, I meant broken.